src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/Constants.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace WcfTestBridgeCommon
+{
+    public static class Constants
+    {
+        public const string CertificateAuthorityName = "DO_NOT_TRUST_WcfBridgeRootCA";
+    }
+}
```

Comment:
Either remove this file, or ensure that you follow the guidance in https://github.com/dotnet/wcf/pull/483/files#r43360563 for usage.